doc/content/reference/configuration/the-things-stack.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Generic Server Options"
+title: "General Options"
 description: ""
-weight: 1
+weight: -1
```

Comment:
What does negative weight mean?

Reply:
It's sorted numerical from lowest to highest. So this basically indicates to show it before others that have a (positive) number (if any)